### PR TITLE
[devtools-translate] Japanese - What's New in DevTools (Chrome 100)

### LINKED ISSUE #2265: [devtools-translate] Japanese - What's New in DevTools (Chrome 100)
Post: https://developer.chrome.com/blog/new-in-devtools-100/

Translator: @yoichiro
Reviewers: @lacolaco, @technohippy, @yoshiko-pg 

Translated by: Mar 22

### PR DESCRIPTION
**Pull request by yoichiro**
Fixes #2265

Changes proposed in this pull request:

Translated "What's New in DevTools 100" into Japanese.

### CODE REVIEW

**Comment by yoichiro** [comment]
@jecfish @agektmr I have just created this pull request to translate the "What's New in DevTool (Chrome 100)" article into Japanese.

@technohippy @lacolaco @yoshiko-pg Could you review this? Thanks!

File: site/ja/blog/new-in-devtools-100/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ authors:
   - jecelynyeen
 date: 2022-03-08
 updated: 2022-03-08
-description: 'View and edit @supports at rules, rename and customize recording’s selector, and more.'
+description: '@supports アットルールの表示と編集, レコーディングのセレクタのリネームやカスタマイズなど'
```
Comment (technohippy) [inline site/ja/blog/new-in-devtools-100/index.md:8]:
```suggestion
description: '@supports アットルールの表示と編集、レコーディングのセレクタのリネームやカスタマイズなど'
```

File: site/ja/blog/new-in-devtools-100/index.md [hunk 2]
```diff
@@ -64,37 +64,47 @@ Chromium issues: [1222574](https://crbug.com/1222574), [1222573](https://crbug.c
...
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/4diI81kpscXznWLrB6a9.png", alt="Support common selectors by default", width="800", height="585" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/4diI81kpscXznWLrB6a9.png", alt="デフォルトで共通セレクタをサポート", width="800", height="585" %}
 
 
 <!-- ### Customize the recording’s selector {: #customize-selector } -->
+### レコーディングのセレクタをカスタマイズする {: #customize-selector }
 
 <!-- You can customize the selector of a recording if you are not using the [common selectors](/docs/devtools/recorder/#selector). -->
+[一般的なセレクタ](/docs/devtools/recorder/#selector) を使用していない場合は、レコーディングのセレクタをカスタマイズすることができます。
```
Comment (technohippy) [inline site/ja/blog/new-in-devtools-100/index.md:82]:
```suggestion
[一般的なセレクタ](/docs/devtools/recorder/#selector) を使用していない場合は、レコーディングのセレクタをカスタマイズできます。
```

**Review by technohippy — COMMENTED** [review]
ちょっとした修正です

**Review by lacolaco — COMMENTED** [review]
LGTM if existing comments were addressed. 👍

File: site/ja/blog/new-in-devtools-100/index.md [hunk 3]
```diff
@@ -64,37 +64,47 @@ Chromium issues: [1222574](https://crbug.com/1222574), [1222573](https://crbug.c
 - data-testing
 
 <!-- The attributes above are common selectors used in test automation.  -->
+上記の属性は、テストの自動化で使用される一般的なセレクタです。
 
 <!-- For example, [start a new recording](/docs/devtools/recorder/#record) with this [demo page](https://jec.fyi/demo/recorder). Fill in an email address and observe the selector value. -->
+例えば、この [デモページ](https://jec.fyi/demo/recorder) で [新規にレコーディングを開始](start a new recording](/docs/devtools/recorder/#record) します。 メールアドレスを入力して、セレクタの値を確認してみてください。
```
Comment (yoshiko-pg) [inline site/ja/blog/new-in-devtools-100/index.md:70]:
```suggestion
例えば、この [デモページ](https://jec.fyi/demo/recorder) で [新規にレコーディングを開始](/docs/devtools/recorder/#record) します。 メールアドレスを入力して、セレクタの値を確認してみてください。
```

**Review by yoshiko-pg — COMMENTED** [review]
翻訳おつかれさまです！1点コメントしました！

**Comment by yoichiro** [comment]
@jecfish @agektmr The review by GDEs has been completed. There is an error about checking CLAs, because I committed the diff provided by Yoshiko and she did not agree the license aggreement at that time. I heard that she has already agreed it, therefore I think there is no problem. Could you go forward to final check and merging this?

**Comment by agektmr** [comment]
Fixed.

**Comment by agektmr** [comment]
Sorry, I'm a bit too busy to review this.

**Review by jecfish — APPROVED** [review]

**Comment by jecfish** [comment]
皆さん、ありがとうございました!

**Comment by yoichiro** [comment]
@agektmr @jecfish Thank you for the final checking and merging!

**Pull request merged**